scripts/push-docker-images

```diff
@@ -62,6 +62,8 @@ if [[ $MANIFEST == "true" ]]; then
   fi
   cat <<< "$(jq '.+{"experimental":"enabled"}' $DOCKER_CLI_CONFIG)" > $DOCKER_CLI_CONFIG
   echo "Enabled experimental CLI features to execute docker manifest commands"
+  # Delete the local version of the manifest so we rely solely on the remote manifest
+  docker manifest rm $IMAGE_REPO:$VERSION || :
```

Comment:
I feel like there should be a warning before deleting user's local manifest or maybe a way to restore it because I wouldn't expect a `push-docker-images` script to delete anything

Reply:
Running this script on a local developer machine is already an edge case to be done with extreme caution. The primary place this will run is in the CI/CD system, where no one is creating manifests by hand anyway.

The local manifest should not be a source of truth at this stage of the process. A local manifest may lack some images that are present in the remote manifest, so it is almost always incorrect to rely on an existing local manifest that was not created by this script.

Unfortunately `docker manifest inspect` doesn't draw a distinction between local manifests and remote manifests, so this is the only way to ensure we rely on the remote manifest as a starting point.

All that said, what kind of warning would you want to see? I'm assuming you're thinking of something that would get the attention of a human user without disrupting automated builds?

Reply:
> Running this script on a local developer machine is already an edge case to be done with extreme caution

Gotcha, thanks for the context. In that case, this is good to go 👍